flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "skip-arches": ["arm"]
```

Comment:
Oh I should have mentioned: `aarch64` is basically 64-bit ARM
```suggestion
  "skip-arches": ["arm", "aarch64"]
```

Reply:
Thanks! That should fix it now.